src/browser/ReactDOM.js

```diff
@@ -84,6 +84,7 @@ var ReactDOM = mapObject({
   h6: 'h6',
   head: 'head',
   header: 'header',
+  hgroup: 'hgroup',
```

Comment:
But `hgroup` was dropped from the spec (which is why I never added it). But I guess if it's still supported, fine.